catalog/policies/plan.warn-on-resource-changes-except-image-id.rego

```diff
@@ -0,0 +1,14 @@
+package spacelift
+
+sample { true }
+
+warn["This is changing more than one resource"] {
+  input.spacelift.run.changes[_].action == "changed"
+  count(input.spacelift.run.changes) > 1
+}
+
+warn["This is changing the aws_launch_template besides its image_id"] {
+  input.spacelift.run.changes[_].action == "changed"
+  not input.spacelift.run.changes[_].entity.type == "aws_launch_template"
```

Comment:
why negate (`not`) here?

Reply:
It says "This is changing the aws_launch_template" but `not input.spacelift.run.changes[_].entity.type == "aws_launch_template"` does the opposite and then AND-ed with the other statements